### PR TITLE
Automation test for issue #475

### PR DESCRIPTION
**Pull request by dkijania**

### CODE REVIEW

File: jormungandr-integration-tests/src/jcli/transaction/id.rs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    let mut builder = JCLITransactionWrapper::new_transaction(FAKE_GENESIS_HASH);
+
+    builder
+        .assert_add_input(&FAKE_INPUT_TRANSACTION_ID, &0, &200)
+        .assert_add_output(&reciever.address, &150)
+        .assert_finalize_with_spending_address(&sender.address);
+
+    let transaction_id_before = builder.get_transaction_id();
+    builder.assert_add_certificate(&FAKE_CERTIFICATE);
```
Comment (vincenthz) [inline jormungandr-integration-tests/src/jcli/transaction/id.rs]: certificate should be added before finalization, most definitely will not work; we had a bug that was requiring the certificate after finalization

File: jcli/.gitignore [hunk 2]
```diff
@@ -0,0 +1 @@
+/target
```
Comment (vincenthz) [inline jcli/.gitignore]: already one at the root that ignore this already, what's the point of this ?

File: jormungandr-integration-tests/src/jcli/transaction/id.rs (hunk 1 above)
Comment (dkijania) [reply jormungandr-integration-tests/src/jcli/transaction/id.rs]: In steps to reproduce for issue #475 transaction is finalized prior to add-certificate, bug was accepted in that form and this is how it's working right now (on master from 14.06).  So `add-certificate` command should change transaction id only before finalize ? and if we use after finalize jcli should complain?

Comment (dkijania) [reply jormungandr-integration-tests/src/jcli/transaction/id.rs]: ok. looks like after rebase it's working as you said

**Comment by NicolasDP** [comment]
#475 is closed now. I am not sure if this is still relevant now.

**Comment by dkijania** [comment]
I think it is. as a regression test

**Comment by dkijania** [comment]
This PR is rejected, based on our new agreement i will implement it in lower layer. probably jcli

**Pull request closed without merging**